app/controllers/fusor/api/v21/lifecycle_environments_controller.rb

```diff
@@ -13,7 +13,27 @@ def show
   end
 
   def create
-    #todo
+    connection = get_sat_connection
+
+    # TODO - remove hardcode of ?organization_id=1
+    json_response = connection.post do |req|
+      req.url 'katello/api/v2/environments?organization_id=1'
+      req.headers['Content-Type'] = 'application/json'
+      req.body = {environment: params[:lifecycle_environment]}.to_json
+    end
+
+    render json: {lifecycle_environment: {
+                      id: json_response.body['id'],
+                      name: json_response.body['name'],
+                      label: json_response.body['label'],
+                      description: json_response.body['description'],
+                      prior: (json_response.body['prior']['id'] if json_response.body['prior']),
+                      prior_id: (json_response.body['prior']['id'] if json_response.body['prior']),
```

Comment:
@cfchase, the json_response from Katello API v2 isn't wrapped with the model_name: as in the Foreman API v2 response.